sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorExporterOptions.cs

```diff
@@ -3,10 +3,15 @@
 
 using Azure.Core;
 
+using System.ComponentModel;
+
 namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor
 {
     public class AzureMonitorExporterOptions : ClientOptions
     {
         public string ConnectionString { get; set; }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string ApiVersion { get; set; }
```

Comment:
minor: consider putting this before `ConnectionString` (sort alphabetically) unless there is a reason not doing it.